docs/content/installation/integrations/app-protect-waf/configuration.md

```diff
@@ -228,6 +228,24 @@ spec:
     apBundle: "<policy_bundle_name>.tgz"
 ```
 
+This example shows the same policy as above but with a log bundle used for :
```

Comment:
@oseoin - is there a missing word here?

Reply:
No idea how I missed this, thanks @jputrino! Fixed in https://github.com/nginxinc/kubernetes-ingress/pull/5269